rest_api/controller/feedback.py

```diff
@@ -11,20 +11,20 @@
 logger = logging.getLogger(__name__)
 
 
-@router.post("/feedback", operation_id="post_feedback")
+@router.post("/feedback")
 def user_feedback(feedback: LabelSerialized):
```

Comment:
When we remove the operation_id we should rename the method accordingly (here: "post_feedback") so that the exported openapi spec doesn't change